enhancements/sig-architecture/224-event-based-manifestwork/README.md

```diff
@@ -497,14 +499,16 @@ sequenceDiagram
 2. Extensions
 ```JSON5
 {
-    // specResourceID is from identifier of the corresponding spec message.
+    // ResourceID is from identifier of the corresponding spec message.
     // It is represented in string.
     // It is a required property.
-    "specResourceID": "<uuid-of-the-resource>",
-    // specResouceVersion is resource version of the corresponding spec message.
+    "ResourceID": "<uuid-of-the-resource>",
+    // ResouceVersion is resource version of the corresponding spec message.
     // It is represented in int64.
     // It is a required property.
-    "specResouceVersion": "<resource-version-in-int64>",
+    "ResouceVersion": "<resource-version-in-int64>",
```

Comment:
should be `resourceID` and `resourceVersion`? @skeeey

Reply:
yes, it should be, and should we use lower case for all attributes in cloud event?